todo/models.py

```diff
@@ -32,7 +33,10 @@ class Task(models.Model):
     completed = models.BooleanField(default=False)
     completed_date = models.DateField(blank=True, null=True)
     created_by = models.ForeignKey(
-        settings.AUTH_USER_MODEL, related_name="todo_created_by", on_delete=models.CASCADE
+        settings.AUTH_USER_MODEL,
+        null=True,
```

Comment:
Hmm, this removes a db constraint for the rest of the app, where created_by should never be null. That's not horrible nor particularly risky as the rest of the code does the right thing, but an alternative would be to assign it to a "system" user (which could be instantiated in a django fixture so it's always present in new installs). OR ... django-todo already provides a setting for `TODO_DEFAULT_ASSIGNEE=username`. That setting was introduced for the purpose of accepting tickets over the web in `external_add`. I think it might make sense to continue having this field be not null, and to rely on that setting when creating your initial tickets. Your documentation could mention that this setting must be present to use the feature. WDYT?

Reply:
I would rather document the fact using this feature creates comments and tasks without a creating user. I don't see this as an issue because the only place tasks / comments are created without a creating user is without the mail worker code. So existing code won't break unless it is used.

>  think it might make sense to continue having this field be not null, and to rely on that setting when creating your initial tickets. 

but... it's not what the name of the setting says :'(

> Your documentation could mention that this setting must be present to use the feature

I would rather mention the mail worker creates tasks / comments without creating user

Reply:
I can live with this.